src/site/xdoc/manual/layouts.xml.vm

```diff
@@ -1455,9 +1455,9 @@ WARN  [main]: Message 2</pre>
             <tr>
               <td align="center">
                 <a name="PatternMessage"/>
-                <b>m</b>{nolookups}{ansi}<br />
-                <b>msg</b>{nolookups}{ansi}<br />
-                <b>message</b>{nolookups}{ansi}
+                <b>m</b>{ookups}{ansi}<br />
```

Comment:
minor: nolookups was changed to lookups below, but here it is "ookups", missing "l"; ignore if this was intended

Reply:
Thanks @mdpollard! This issue is addressed in more recent commits.